### ISSUE TITLE
cassandra crash when using python driver

### ISSUE THREAD

**Issue by lapcchan** [Stratio/cassandra-lucene-index#123]
for 3.0.4 or 3.0.5, while it works fine on cqlsh, when using python cassandra driver 3.2.2 or 3.0.0 it will crash.

mostly any query will crash, for eg: SELECT * FROM tweets WHERE expr(tweets_index,'{ refresh : true}');

i am using python-2.7.5-34.el7.x86_64 the one from centos7 and java-1.8.0-openjdk-1.8.0.91-0.b14.el7_2.x86_64

python logs:
```
<class 'cassandra.ReadFailure'> code=1300 [Replica(s) failed to execute read] message="Operation failed - received 0 responses and 1 failures" info={'failures': 1, 'received_responses': 0, 'required_responses': 1, 'consistency': 'ONE'}
```

cassandra logs:
```
DEBUG [SharedPool-Worker-1] 2016-04-29 13:59:04,403 ReadCallback.java:126 - Failed; received 0 of 1 responses
ERROR [SharedPool-Worker-2] 2016-04-29 13:59:04,418 JVMStabilityInspector.java:139 - JVM state determined to be unstable.  Exiting forcefully due to:
java.lang.OutOfMemoryError: Requested array size exceeds VM limit
        at java.util.ArrayList.<init>(ArrayList.java:152) ~[na:1.8.0_91]
        at com.stratio.cassandra.lucene.index.FSIndex.lambda$doSearch$0(FSIndex.java:288) ~[cassandra-lucene-index-plugin-3.0.4.1.jar:na]
        at com.stratio.cassandra.lucene.index.FSIndex$$Lambda$184/145226602.apply(Unknown Source) ~[na:na]
        at com.stratio.cassandra.lucene.index.FSIndex.doWithSearcher(FSIndex.java:147) ~[cassandra-lucene-index-plugin-3.0.4.1.jar:na]
        at com.stratio.cassandra.lucene.index.FSIndex.doSearch(FSIndex.java:278) ~[cassandra-lucene-index-plugin-3.0.4.1.jar:na]
        at com.stratio.cassandra.lucene.index.DocumentIterator.fetch(DocumentIterator.java:68) ~[cassandra-lucene-index-plugin-3.0.4.1.jar:na]
        at com.stratio.cassandra.lucene.index.DocumentIterator.hasNext(DocumentIterator.java:87) ~[cassandra-lucene-index-plugin-3.0.4.1.jar:na]
        at com.stratio.cassandra.lucene.IndexReaderSkinny.prepareNext(IndexReaderSkinny.java:58) ~[cassandra-lucene-index-plugin-3.0.4.1.jar:na]
        at com.stratio.cassandra.lucene.IndexReader.hasNext(IndexReader.java:68) ~[cassandra-lucene-index-plugin-3.0.4.1.jar:na]
        at org.apache.cassandra.db.transform.BasePartitions.hasNext(BasePartitions.java:72) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.db.partitions.UnfilteredPartitionIterators$Serializer.serialize(UnfilteredPartitionIterators.java:295) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.db.ReadResponse$LocalDataResponse.build(ReadResponse.java:134) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.db.ReadResponse$LocalDataResponse.<init>(ReadResponse.java:127) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.db.ReadResponse$LocalDataResponse.<init>(ReadResponse.java:123) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.db.ReadResponse.createDataResponse(ReadResponse.java:65) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.db.ReadCommand.createResponse(ReadCommand.java:289) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.service.StorageProxy$LocalReadRunnable.runMayThrow(StorageProxy.java:1792) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.service.StorageProxy$DroppableRunnable.run(StorageProxy.java:2446) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_91]
        at org.apache.cassandra.concurrent.AbstractLocalAwareExecutorService$FutureTask.run(AbstractLocalAwareExecutorService.java:164) ~[apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.concurrent.AbstractLocalAwareExecutorService$LocalSessionFutureTask.run(AbstractLocalAwareExecutorService.java:136) [apache-cassandra-3.0.4.jar:3.0.4]
        at org.apache.cassandra.concurrent.SEPWorker.run(SEPWorker.java:105) [apache-cassandra-3.0.4.jar:3.0.4]
        at java.lang.Thread.run(Thread.java:745) [na:1.8.0_91]
```

**Comment by lapcchan** [comment]
its the same problem as #124 
python driver with default_fetch_size = None will crash any query without limit.

**Comment by ealonsodb** [comment]
Hi @lapcchan , When using default_fetch_Size=None with python driver efectively it passes the fetch size equal to Integer.MAX_VALUE. This cause this error trying to allocate an array with that size. 
I think that we fixed it. 

Can you help us and try to execute the query with last code in [branch-3.0.5](https://github.com/Stratio/cassandra-lucene-index/tree/branch-3.0.5)

Thanks

**Comment by adelapena** [comment]
This is fixed  in [3.0.5.2](https://github.com/Stratio/cassandra-lucene-index/tree/3.0.5.2), [3.5.2](https://github.com/Stratio/cassandra-lucene-index/tree/3.5.2) and [3.0.6.0](https://github.com/Stratio/cassandra-lucene-index/tree/3.0.6.0) releases.

Thanks for reporting.
